BUILDING.md

```diff
@@ -8,30 +8,26 @@ Session uses [Gradle](http://gradle.org) to build the project and to maintain
 dependencies.  However, you needn't install it yourself; the
 "gradle wrapper" `gradlew`, mentioned below, will do that for you.
 
-Building Session
+Dependencies
 ---------------
+You will need Java 8 set up on your machine.
 
-The following steps should help you (re)build Session from the command line.
-
-1. Checkout the session-android project source with the command:
-
-        git clone https://github.com/loki-project/session-android.git
-
-2. Make sure you have the [Android SDK](https://developer.android.com/sdk/index.html) installed.
-3. Ensure that the following packages are installed from the Android SDK manager:
+Ensure that the following packages are installed from the Android SDK manager:
     * Android SDK Build Tools (see buildToolsVersion in build.gradle)
     * SDK Platform (All API levels)
     * Android Support Repository
     * Google Repository
-4. Create a local.properties file at the root of your source checkout and add an sdk.dir entry to it.  For example:
+    
+In Android studio, this can be done from the Quickstart panel, choose "Configure" then "SDK Manager". In the SDK Tools tab of the SDK Manager, make sure that the "Android Support Repository" is installed, and that the latest "Android SDK build-tools" are installed. Click "OK" to return to the Quickstart panel. You may also need to install API version 28 in the SDK platforms tab.
+    
+You will then need to clone and run `./gradlew build` on each of the following repositories IN ORDER.
 
-        sdk.dir=/Application/android-sdk-macosx
-
-5. Using Java 8 
-
-6. Execute Gradle:
+https://github.com/loki-project/loki-messenger-android-curve-25519
+https://github.com/loki-project/loki-messenger-android-protocol
+https://github.com/loki-project/loki-messenger-android-meta
```

Comment:
Actually the `protocol` and `meta` repos have been merged into `service` so we don't need to clone them

Reply:
https://github.com/loki-project/session-android/pull/201/commits/e653a21e37fc2276b17083961c53292f925978f8